probe/kubernetes/persistentvolume.go

```diff
@@ -45,13 +48,31 @@ func (p *persistentVolume) GetVolume() string {
 	return volume
 }
 
+// GetStorageDriver returns the backing driver of Persistent Volume
+func (p *persistentVolume) GetStorageDriver() string {
```

Comment:
Are you sure that the k8s library doesn't supply a way to get the textual representation of the driver?

Reply:
(if not, then that's OK, we could use something like https://github.com/fatih/structs but it's completely overkill for just this)

Reply:
@2opremio AFAIK, k8s library doesn't provide the textual representation of the driver directly, 
and yes, we can use https://github.com/fatih/structs but I don't want to introduce any dependency for such a small change.